beginner_source/ddp_series_fault_tolerance.rst

```diff
@@ -42,8 +42,8 @@ Follow along with the video below or on `youtube <https://www.youtube.com/watch/
 
 In distributed training, a single process failure can
 disrupt the entire training job. Since the susceptibility for failure can be higher here, making your training
-script robust is particularly important here. You might also prefer your training job to be *elastic* i.e. 
-
+script robust is particularly important here. You might also prefer your training job to be *elastic*, i.e.,
```

Comment:
```suggestion
script robust is particularly important here. You might also prefer your training job to be *elastic*, for example,
```

Reply:
I agree, this is more clear! Commit incoming